Intellisense not working for inline dependencies

Hi, 
I've been reloading visual studio code many times, I'm trying to do as in the doc (https://github.com/filipw/dotnet-script#nuget-packages) but intellisense would not work with the following piece of code :
```
#! "netcoreapp2.0"

#r "nuget: Newtonsoft.Json,11.0.1"
#load "simpleHttpRequest.csx"

using System.Linq;
using Newtonsoft.Json.Linq;
```
The script works at runtime but in Visual Studio code I have Newtonsoft underlined in red, and no intellisense. (the same as for all JObject uses etc.)

VS Code 1.20.1
ms-vscode.csharp : 1.14.0
dotnet.script : 0.18.0

Sure : 
[omnisharp-detailed.txt](https://github.com/filipw/dotnet-script/files/1780921/omnisharp-detailed.txt)

Looks like a problem with the "under the hood" package restoration... 

```
[dbug]: OmniSharp.Script.Dotnet.Script.DependencyModel.Process.CommandRunner
        2.1.4
[dbug]: OmniSharp.Script.Dotnet.Script.DependencyModel.Context.DotnetRestorer
        Restoring C:\Users\vincent.labatut\AppData\Local\Temp\scripts\d\Lab\Script2\script.csproj using the dotnet cli.
[dbug]: OmniSharp.Script.Dotnet.Script.DependencyModel.Process.CommandRunner
        C:\Users\vincent.labatut\AppData\Local\Temp\scripts\d\Lab\Script2\script.csproj : error MSB4057: The target "Restore" does not exist in the project.
```
script.csproj is :
```
<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
  </ItemGroup>
</Project>
```

Reproduced !
Overwriting all the env variables with the ones from the dotnet process launched by VSCode and i have the bug in my command line :
```
D:\Lab\Script2>"dotnet.exe" exec "C:\Program Files\dotnet\sdk\2.1.4\MSBuild.dll" /m /v:m /nologo /t:Restore C:\Users\vincent.labatut\AppData\Local\Temp\scripts\d\Lab\Script2\script.csproj "/Logger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\2.1.4\dotnet.dll"
C:\Program Files\dotnet\sdk\2.1.4\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props(29,3): error MSB4019: The imported project "C:\Users\vincent.labatut\.vscode\extensions\ms-vscode.csharp-1.14.0\.omnisharp\msbuild\15.0\Bin\15.0\Microsoft.Common.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\vincent.labatut\AppData\Local\Temp\scripts\d\Lab\Script2\script.csproj]
```

--- a/src/Dotnet.Script.DependencyModel/Context/DotnetRestorer.cs
+++ b/src/Dotnet.Script.DependencyModel/Context/DotnetRestorer.cs
@@ -18,8 +18,8 @@ namespace Dotnet.Script.DependencyModel.Context
 
         public void Restore(string pathToProjectFile)
         {
-            _logger.Debug($"Restoring {pathToProjectFile} using the dotnet cli.");
             var runtimeIdentifier = RuntimeHelper.GetRuntimeIdentifier();
+            _logger.Debug($"Restoring {pathToProjectFile} using the dotnet cli. RuntimeIdentifier : {runtimeIdentifier}");            
             var exitcode = _commandRunner.Execute("dotnet", $"restore \"{pathToProjectFile}\" -r {runtimeIdentifier}");
             if (exitcode != 0)
             {

--- a/src/Dotnet.Script.DependencyModel/Dotnet.Script.DependencyModel.csproj
+++ b/src/Dotnet.Script.DependencyModel/Dotnet.Script.DependencyModel.csproj
@@ -11,7 +11,7 @@
     <RepositoryUrl>https://github.com/filipw/dotnet-script.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageTags>script;csx;csharp;roslyn;omnisharp</PackageTags>
-    <Version>0.4.0</Version>    
+    <Version>0.5.0</Version>    
   </PropertyGroup>
 
   <ItemGroup>

--- a/src/Dotnet.Script.DependencyModel/Process/CommandRunner.cs
+++ b/src/Dotnet.Script.DependencyModel/Process/CommandRunner.cs
@@ -1,4 +1,5 @@
-﻿using System.Diagnostics;
+﻿using System.Collections.Generic;
+using System.Diagnostics;
 using Dotnet.Script.DependencyModel.Logging;
 
 namespace Dotnet.Script.DependencyModel.Process
@@ -27,11 +28,20 @@ namespace Dotnet.Script.DependencyModel.Process
                 CreateNoWindow = true,
                 Arguments = arguments ?? "",
                 RedirectStandardOutput = true,
-                RedirectStandardError = true,
+                RedirectStandardError = true,                
                 UseShellExecute = false
             };
+            RemoveMsBuildEnvironmentVariables(startInformation.Environment);
             return startInformation;
         }
+        private static void RemoveMsBuildEnvironmentVariables(IDictionary<string, string> environment)
+        {
+            // Remove various MSBuild environment variables set by OmniSharp to ensure that
+            // the .NET CLI is not launched with the wrong values.
+            environment.Remove("MSBUILD_EXE_PATH");
+            environment.Remove("MSBuildExtensionsPath");
+        }
+
 
         private static void RunAndWait(System.Diagnostics.Process process)
         {

